internal/pkg/pct/install.go

```diff
@@ -111,3 +149,35 @@ func (p *PctInstaller) setupTemplateNamespace(targetDir string, info PuppetConte
 
 	return namespacePath, err
 }
+
+func (p *PctInstaller) downloadTemplate(targetURL *url.URL, downloadDir string) (string, error) {
+	// Get the file contents from URL
+	response, err := Client.Get(targetURL.String())
+	if err != nil {
+		return "", nil
```

Comment:
Should would be returning a `nil` error here? This would then mean the [calling code above](https://github.com/puppetlabs/pdkgo/pull/203/files#diff-41591865107d2fbc5df365ab70965744fb07a0f8e8c54a877dce9fb3962a04e7R60-R62) would assume the download was successful.

Reply:
You're correct, this is a goof!